AWS/jenkins-deployment.md

```diff
@@ -0,0 +1,144 @@
+# (AWS) Jenkins Deployment
+
+<br />
+### Deploy Jenkins via Packer:  
```

Comment:
Add one more section here that explains how you should configure your user variables in baseJenkinsEC2.json